terraform/sms.tf

```diff
@@ -1,31 +0,0 @@
-# Copyright 2020 Google LLC
```

Comment:
We still need this. It's creating an _empty_ secret and granting the required IAM permissions.